<launch>

  <arg name="dir_ims" default="/media/Data/Sign Language Pose Estimation/tracking-groundtruth-sequences/01April_2010_Thursday_heute_default-0/"/>

  <node pkg="faceTracking" type="faceTrack" name="flow" output="screen">
	<remap from="image" to="/camera/image"/>
  </node>

  <node pkg="handBlobTracker" type="handTracker" name="handTracker" output="screen">
	<remap from="/rgb/image_color" to="/camera/image"/>
  </node>

  <node pkg="mkfbodytracker_pdaf" type="poseTracker" name="poseTracker" output="screen">
	<remap from="/rgb/image_raw" to="/camera/image"/>
  </node>

  <node pkg="bag_tools" type="image_sequence_publisher.py" name="image_seq" output="screen" required="true">
	<param name="image_dir" value="$(arg dir_ims)"/>
	<param name="file_pattern" value="*.png"/>
        <param name="camera_info_file" value="/home/mgb45/Documents/ros_workspace/mkfbodytracker/cal_sign.yml"/>
        <param name="frequency" value="10"/>
  </node>

  <node pkg="image_view" type="image_view" name ="im2">
	<remap from="image" to ="outputImage"/>
  </node>

  <node pkg="image_view" type="image_view" name ="im3">
        <remap from="image" to ="likelihood"/>
  </node>

  <node pkg="image_view" type="image_view" name ="im4">
        <remap from="image" to ="poseImage"/>
  </node>

</launch>
